src/components/index.d.ts

```diff
@@ -1308,6 +1308,46 @@ export declare const NoraCheckboxInput: {
     props: any
   }
 }
+export declare const NoraButtonInput: {
+  ({
+    iconPrefix,
+    iconName,
+    side,
+    buttonDisabled,
+    // NoraTextInput props are passed through
```

Comment:
The 'pass through' parameters for `TextInput` - is there a need to enumerate them one by one or can we do a spread of the props to the `TextInput`, similar to how you've done in other derived input components.

Reply:
In this d.ts file, my thought was that explicitly defining these would provide the type hinting in Typescript and our IDE's.

Reply:
The more type hinting, the better :)